lib/get_parser.py

```diff
@@ -280,6 +280,12 @@ def get_parser() -> argparse.ArgumentParser:
         help="generate and mail a summary report of completed/failed/removed"
         " jobs in a DAG",
     )
+    parser.add_argument(
+        "--job-info",
+        action="append",
+        default=[],
+        help="script to call with jobid, etc. when job is launched",
```

Comment:
Is this run when the job is launched or when it is submitted?  From `condor.py`, it looks like the latter.

Reply:
So I reworded it... (I've always used launched and submitted interchangably, but I can see where someone would think launched was when the first job actually started...)